kura/org.eclipse.kura.wire.component.provider/OSGI-INF/metatype/org.eclipse.kura.wire.Timer.xml

```diff
@@ -29,7 +29,7 @@
             cardinality="0"
             required="true"
             default="10"
-            description="Interval of the timer ticks, in the time unit specified by the simple.time.unit property (Needs to be set if SIMPLE type is selected)"
+            description="Interval of the timer ticks, in the time unit specified by the simple.time.unit property. This value is used when type is set to SIMPLE."
```

Comment:
Change description is OK, please update the header of the document.

Reply:
Done